_posts/2019-12-20-announcing-the-privacytools-sponsorship.md

```diff
@@ -15,13 +15,13 @@ We are delighted to be in a position where we are able to share our resources, a
 
 ## Their Work Impacted Us Personally
 
-Before the Safing venture started, the work of PrivacyTools already impacted all three of us founders. Their website helped us in numerous ways. Whether to deepen our understanding of the importance of privacy or about arguments we can make for it. And most importantly, their software recommendations and instructions armed us to better fight today's mass surveillance. I fondly remember the first time I proudly configured my `about:config` of Firefox, enthusiastically heading to the next section to install additional privacy enhancing extensions.
+Before the Safing journey started, the work of PrivacyTools already impacted all three of us founders. Their website helped us in numerous ways. Whether to deepen our understanding of the importance of privacy or about arguments we can make for it. And most importantly, their software recommendations and instructions armed us to better fight today's mass surveillance. I fondly remember the first time I proudly configured my `about:config` of Firefox, enthusiastically heading to the next section to install additional privacy enhancing extensions.
```

Comment:
we are getting to big to call it "us founders" I would rather go with "the founders"

Reply:
but the personal note afterwards still is great, so maybe you want the us in there?